src/helpers/curve.ts

```diff
@@ -119,19 +63,18 @@ export function getAmountAFromLiquidityDelta(
   maxSqrtPrice: BN,
   rounding: Rounding
 ): string {
```

Comment:
Why not return BN?

Reply:
It was my mistake to keep the old behavior when I used Decimal in the previous version. I've updated it.